README.md

```diff
@@ -92,7 +92,7 @@ These APIs are available at `http://connection-apache/` endpoint from clients te
 Before executing tests please install dev dependencies:
 - Prepare and export AWS keys for [Snowflake drivers download](https://github.com/keboola/drivers-management#driver-download) required by build
 - `docker-compose build`
-- `docker-compose run --rm tests composer install`
+- `docker-compose run --rm dev composer install`
```

Comment:
I think there may be other places to update this in this readme too.  (line 100 for ex)

Reply:
Ok :) I will fix that

Reply:
[https://github.com/keboola/storage-api-php-client/pull/318/commits/e0707b686a56acf8382176a522048cb3a35aa141](url)